modules/util.py

```diff
@@ -406,6 +406,19 @@ def startUdevDaemon():
         return False
     return True
 
+def getCurrentHostOSVersion(conffile):
+    ''' Get current OS version from /etc/os-release '''
+    root_mount = getConfigurationItem(conffile, 'General', 'host_bind_mount')
+    if not root_mount:
+        root_mount = '/'
+    if os.path.isfile(os.path.join(root_mount, "etc/os-release")):
+        with open(os.path.join(root_mount, "etc/os-release"), 'r') as f:
```

Comment:
This has a TOCTOU problem. It's better if you try to open the file for reading and put a try/catch block to catch the exception. Currently it can crash the updater